src/authentication/authentication.api.ts

```diff
@@ -12,3 +12,19 @@ export function callForLogin(): Promise<unknown> {
 export async function callForLogout(): Promise<unknown> {
   return tabNineProcess.request({ Logout: {} });
 }
+
+export async function signInUsingCustomToken(
+  customToken: string
+): Promise<unknown> {
+  return tabNineProcess.request({
+    LoginWithCustomToken: { custom_token: customToken },
+  });
+}
+
+export async function signInUsingCustomTokenUrl(): Promise<
```

Comment:
can this return an error? how will we debug it? is the binary logging it for us?